plugins/ocm-backend/config.d.ts

```diff
@@ -0,0 +1,40 @@
+interface KubernetesPluginRef {
+  /**
+   * Match cluster name where the Open Cluster Management operator is installed from the kubernetes plugin configuration
+   */
+  kubernetesPluginRef: string;
+}
+
+interface HubClusterConfig {
+  /**
+   * Url of the hub cluster API endpoint
+   */
+  url: string;
+  /**
+   * Name of the cluster
```

Comment:
I would change this to a something more descriptive, maybe the same description as in the README example configuration.